st2common/st2common/util/sandboxing.py

```diff
@@ -23,14 +23,20 @@
 import fnmatch
 import os
 import sys
-from distutils.sysconfig import get_python_lib
+from sysconfig import get_path
 
 from oslo_config import cfg
 
 from st2common.constants.action import LIBS_DIR as ACTION_LIBS_DIR
 from st2common.constants.pack import SYSTEM_PACK_NAMES
 from st2common.content.utils import get_pack_base_path
 
+
+def get_python_lib():
+    """Replacement for distutil.sysconfig.get_python_lib, returns a string with the python platform lib path (to site-packages)"""
+    return get_path("platlib")
```

Comment:
Just for reference: https://bugs.python.org/issue41282#msg393018 how it's a replacement for `distutil.sysconfig.get_python_lib`.